views/tracks.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="container-collection">
+{{#each tracks}}
+    <div class="container-element">
+        <h3>
+            {{name}}
+        </h3>
+        {{#if uri}}
+        <audio src="{{uri}}"></audio>
```

Comment:
- Add "controls" attribute to have play/pause button displayed
- Instead of using "uri", you should use "preview_url"